lightblue-ldap-integration-test/src/test/java/com/redhat/lightblue/crud/ldap/ITCaseLdapCRUDController_InvalidMetadata_Test.java

```diff
@@ -65,7 +65,7 @@ public void testMetadata_WithoutUniqueFieldDefined() throws Exception {
 
         //Remove the uid field from the fields definition
         String metadataWithoutUniqueField = AbstractJsonNodeTest.loadResource("./metadata/person-metadata.json").replaceFirst("\"uid\": \\{\"type\": \"string\"\\},", "");
-        JsonNode nodeWithoutUniqueField = json(metadataWithoutUniqueField);
+        JsonNode nodeWithoutUniqueField = json(metadataWithoutUniqueField, true);
```

Comment:
Yes, I use substitution in the metadata. I use this for database and basedn values.